### DISCUSSION TITLE
Hosts file

### DISCUSSION THREAD

**Discussion by Buranich** [SpotX-Official/SpotX#232] [Question] [answered]
Hello, I installed Spot-X, it edited the hosts file, deleting everything that was in it, but at the same time created a backup copy somewhere, but i don't know where. The question: where it can be?

**Comment by amd64fox** [comment]
the backup copy is located in the same folder as the original hosts file, named hosts.bak

**Comment by Buranich** [comment, marked as answer]
It's strange but .bak file is missing even though the console says it was created.

**Reply by amd64fox** [reply]
Check what you are looking for in the right folder
- `Win` + `R`
- `%WinDir%\System32\Drivers\Etc\`

**Comment by Buranich** [comment]
I understand where it should be, but **.bak** file is not there, there is only a **empty hosts** file.

**Reply by amd64fox** [reply]
Above I wrote where to look.
do not produce new discussion threads, write in one

if it is not there, then there may have been an error, or your antivirus removed it, it's hard to say
